go/vt/vttablet/tabletserver/throttle/throttler.go

```diff
@@ -294,17 +292,20 @@ func (throttler *Throttler) readThrottlerConfig(ctx context.Context) (*topodatap
 }
 
 // normalizeThrottlerConfig noramlizes missing throttler config information, as needed.
-func (throttler *Throttler) normalizeThrottlerConfig(thottlerConfig *topodatapb.ThrottlerConfig) *topodatapb.ThrottlerConfig {
-	if thottlerConfig == nil {
-		thottlerConfig = &topodatapb.ThrottlerConfig{}
+func (throttler *Throttler) normalizeThrottlerConfig(throttlerConfig *topodatapb.ThrottlerConfig) *topodatapb.ThrottlerConfig {
```

Comment:
IMO this is probably also a good place to validate the app name to be sure that it's valid.

Reply:
commented in https://github.com/vitessio/vitess/pull/13351#discussion_r1253921009, but technically any arbitrary string is valid...